erroranalysis/tests/test_cohort_filter.py

```diff
@@ -355,6 +376,19 @@ def create_iris_pandas():
     return X_train, X_test, y_train, y_test, feature_names
 
 
+def create_iris_str_y():
```

Comment:
could you reuse the create_iris_pandas? Seems like duplicate code here. Maybe pass in a boolean flag in that function to create string labels.

Reply:
done!